윤건수/programmers/LV2_92342/Solution.java

```diff
@@ -0,0 +1,87 @@
+package programmers.LV2_92342;
+
+import java.util.*;
+
+public class Solution {
+
+    final int totalPoint = 10;
+    int totalArrows;
+    int maxPointDiff;
+    int[] apeachArrows;
+    PriorityQueue<boolean[]> rionWinCase;
+
+    public int[] solution(int n, int[] info) {
+        this.totalArrows = n;
+        this.apeachArrows = info;
+        this.maxPointDiff = 0;
+        rionWinCase = new PriorityQueue<>((case1, case2) -> {
+            for(int i = case1.length - 2; i >= 0; i--) {
+                if(case1[i] != case2[i] && case1[i]){
```

Comment:
```suggestion
                if(!case2[i] && case1[i]){
```

case1[i] 가 true 로 고정이면서 case2[i] 랑 같으면 안되니까 case2[i] 는 false 로 설정하면 될 것 같네요.